contracts/DEX.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract DEX {
+    event CDTToSSVConverted(uint256 amount);
+
+    IERC20 public cdtToken;
+    IERC20 public ssvToken;
+
+    uint public rate;
+
+    function init(address _cdtTokenAddress, address _ssvTokenAddress) public {
```

Comment:
you can change the parameters to be of type `IERC20` and then you won't need to cast in the following lines.

Reply:
done